src/fluree/db/storage/proto.cljc

```diff
@@ -2,6 +2,7 @@
   (:refer-clojure :exclude [read list exists?]))
 
 (defprotocol Store
+  (address [store k] "Returns the address that would be constructed by writing to `k`.")
```

Comment:
What is this method used for?

Reply:
Ok. I haven't looked in to the nameservice implementation recently, but It doesn't seem right to me at first blush that something would need to know from the storage implementation what the address will be as a result of a given write before the fact, especially since it's very hard (last i checked impossible) to know this for IPFS without actually writing the file.